packages/compas-convert/src/passes/convert-test-config.ts

```diff
@@ -0,0 +1,49 @@
+import path from "node:path";
+import type { Context } from "../context.js";
+import { getTypescriptProgram } from "./init-ts-morph.js";
+
+/**
+ * Apply .env loading in the test setup + rename methods
+ */
+export async function convertTestConfig(context: Context) {
+	const configFilePath = path.join(context.outputDirectory, "test/config.ts");
+
+	const program = getTypescriptProgram(context);
+	const configFile = program.getSourceFile(configFilePath);
+
+	if (configFile) {
+		configFile.replaceWithText(`
+import { readFileSync } from "node:fs";
+import { parseEnv } from "node:util";
+import { refreshEnvironmentCache, loggerSetGlobalDestination } from "@compas/stdlib";
+
+await setupTestEnvironment();
```

Comment:
Setup files should auto-run, so this automatically injects everything when imported.